modules/update-upgrading-cli.adoc

```diff
@@ -72,7 +72,10 @@ $ oc adm upgrade
 ----
 Cluster version is 4.9.23
 
-Updates:
+Upstream: https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/graph
+Channel: stable-4.10
```

Comment:
There should be available channels after `stable-4.10`, due to v4.9 edges is not added into `stable-4.10` channel yet, so following should be a possible example channels info:
```
Channel: stable-4.10 (available channels: candidate-4.9,fast-4.9,stable-4.9,candidate-4.10,fast-4.10,stable-4.10,eus-4.10)
```

Reply:
Done!